update/process/metaBlock.go

```diff
@@ -98,9 +98,10 @@ func (m *metaBlockCreator) CreateNewBlock(
 		SoftwareVersion:        []byte(""),
 		AccumulatedFees:        big.NewInt(0),
 		AccumulatedFeesInEpoch: big.NewInt(0),
+		DeveloperFees:          big.NewInt(0),
```

Comment:
DevFees or rename the next one DeveloperFeesInEpoch

Reply:
This was not part of this PR